control_center.yml

```diff
@@ -8,18 +8,16 @@
     - import_role:
         name: confluent.variables
 
-    - name: Check if Control Center Service Running
-      shell: "systemctl show -p SubState {{control_center_service_name}}"
-      changed_when: false
-      # On some SELinux enabled hosts this command will error out, handled in next set_fact task
-      failed_when: false
-      check_mode: false
-      register: substate
+    - name: Populate service facts
+      service_facts:
 
-    - set_fact:
-        install_pattern: "{{ 'parallel' if substate.stdout != 'SubState=running' or substate.rc == 1 or control_center_deployment_strategy == 'parallel' else 'serial' }}"
+    - name: "Determine Installation Pattern: Parallel or Serial"
```

Comment:
Does this need to be in quotes?

Reply:
maybe make it 
```suggestion
    - name: Determine Installation Pattern- Parallel or Serial
```

Reply:
Changed to `-` as separation character, and removed the quotes. Nice catch/fix!